src/open_source_python_template/templates/index.html

```diff
@@ -43,7 +43,7 @@ <h2>Starred</h2>
                 </div>
                 <button class="dropdown-toggle" onclick="toggleDropdown('task-detail-{{task['id']}}')">▼</button>
                 <div class="task-detail" id="task-detail-{{task['id']}}">
-                    <p>More details about Task Item...</p>
+                    <p>{{task['taskdescription']}}</p>
```

Comment:
Is this task description optional?

Reply:
yes! can you refer to this previous PR to see how the optional feature is handled: https://github.com/nyuoss/glaceon-google-suite-tasks/pull/19/files